utils/doclint/documentation.js

```diff
@@ -866,6 +866,7 @@ function csharpOptionOverloadSuffix(option, type) {
     case 'Buffer': return 'Byte';
     case 'Serializable': return 'Object';
     case 'int': return 'Int';
+    case 'long': return 'Int64';
```

Comment:
```suggestion
    case 'long': return 'long';
```

I think it should be long? https://learn.microsoft.com/en-us/dotnet/api/system.datetimeoffset.fromunixtimeseconds?view=net-8.0

Reply:
It is for nullable types AFAIU, corresponding boxed type is Int64 similar to Int above, no?